integration/tests/caching/credential_caching_test.go

```diff
@@ -0,0 +1,102 @@
+//go:build integration
+// +build integration
+
+package caching
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/gexec"
+
+	. "github.com/weaveworks/eksctl/integration/runner"
+	"github.com/weaveworks/eksctl/integration/tests"
+	"github.com/weaveworks/eksctl/pkg/eks"
+	"github.com/weaveworks/eksctl/pkg/testutils"
+)
+
+var params *tests.Params
+
+func init() {
+	// Call testing.Init() prior to tests.NewParams(), as otherwise -test.* will not be recognised. See also: https://golang.org/doc/go1.13#testing
+	testing.Init()
+	params = tests.NewParams("cache")
+}
+
+func TestCredentialsCaching(t *testing.T) {
+	testutils.RegisterAndRun(t)
+}
+
+var _ = Describe("", func() {
+	var (
+		clusterName string
+	)
+	BeforeSuite(func() {
+		clusterName = params.NewClusterName("cache")
+		cmd := params.EksctlCreateCmd.WithArgs(
+			"cluster",
+			"--name", clusterName,
+			"--without-nodegroup",
+		).WithoutArg("--region", params.Region)
+		Expect(cmd).Should(RunSuccessfully())
```

Comment:
do we need a cluster? Could just running `eksctl get clusters` be enough?

Reply:
Maaaaybe. 🤔

Reply:
Yeah, you're probably right. For the sake of the test a simple call to get should be enough, doesn't need a cluster for that. :)